tests/api2/test_008_pool_dataset_quota_alert.py

```diff
@@ -78,10 +68,7 @@ def test_dataset_quota_alert(request, datasets, expected_alerts):
         results = SSH_TEST("midclt call --job core.bulk alert.process_alerts '[[]]'", user, password)
```

Comment:
There is no reason to do `SSH_TEST` and do `midclt call` here. These can be done using `call(`